content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -331,6 +331,35 @@ For example:
 Accept: application/vnd.kubernetes.protobuf, application/json
 ```
 
+### CBOR resource encoding {#cbor-encoding}
+
+{{< feature-state feature_gate_name="CBORServingAndStorage" >}}
+
+With the `CBORServingAndStorage` [feature
+gate](/docs/reference/command-line-tools-reference/feature-gates/) enabled, request and response
+bodies for all built-in resource types and all resources defined by a {{< glossary_tooltip
+term_id="CustomResourceDefinition" text="CustomResourceDefinition" >}} may be encoded to the
+[CBOR](https://www.rfc-editor.org/rfc/rfc8949) binary data format. CBOR is also be supported at the
+{{< glossary_tooltip text="aggregation layer" term_id="aggregation-layer" >}} if it is enabled in
+individual aggregated API servers.
+
+Clients should indicate the IANA media type `application/cbor` in the `Content-Type` HTTP request
```

Comment:
```suggestion
Clients should indicate the IANA media type `application/cbor` in the `Content-Type` HTTP request
```

do all the parameter suffixes work as well (e.g. `;as=PartialObjectMetadata;g=meta.k8s.io;v=v1` or `;as=Table;v=v1;g=meta.k8s.io`)?

Reply:
PartialObjectMetadata works today, but I missed a necessary update to EndpointRestrictions here to permit CBOR for tables: https://github.com/kubernetes/kubernetes/blob/c9092f69fc0c099062dd23cd6ee226bcd52ec790/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go#L148. I'll track fixing that.